src/with-font-loading.js

```diff
@@ -17,10 +26,16 @@ withFontLoading('Lato-Bold')(styled('div', props => props.$fontStyles));
 All requested fonts should be defined in src/fonts/fontConfig.js
 */
 
-const withFontLoading = fontName => {
-  return OriginalComponent => {
-    class WithFontLoading extends Component {
-      constructor(props, context) {
+const withFontLoading = (fontName: string) => {
+  return (
+    OriginalComponent: React.ComponentType<*>
+  ): React.ComponentType<*> => {
+    class WithFontLoading extends React.Component<*> {
+      props: *;
+      context: *;
+      state: *;
```

Comment:
State is all local to this component, right? Would it be possible to type this?

Reply:
Yeah, in theory all three of these wouldn't be too difficult to type.  For now, I focused on the signatures of the exported functions, plugin, and tokens.  We'd likely revisit this as a broader push to reach 100% coverage in all of these plugins.